SPECS/llvm/llvm.spec

```diff
@@ -12,10 +12,7 @@ BuildRequires:  cmake
 BuildRequires:  libffi-devel
 BuildRequires:  libxml2-devel
 BuildRequires:  ninja-build
-BuildRequires:  python2
-%if %{with_check}
-BuildRequires:  python-xml
```

Comment:
Does that mean we don't need `python3-xml` or is it already dragged into the build as a dependency for another BR?

Reply:
When I was upgrading our `python3` package to 3.9, I went ahead and folded in `python3-xml` subpackage into the `python3-libs` subpackage. So, it's dragged in through the BR on `python3-devel`